### DISCUSSION TITLE
Feature Request: "Time Seeded" column or ability to create a custom filter that can filter torrents that have been seeded above / below a threshold of time (hours)

### DISCUSSION THREAD

**Discussion by 0utlawActual** [jesec/flood#723] [Ideas]
Hello all,

Really enjoying the look and feel of Flood. One feature  I feel like is missing but would be a great asset would be the ability to either see in a column Time Seeded (say in days/hours) or have the ability to create a custom filter that can inlcude/ exclude torrents that have seeded for x amount of time. I couldn't find asuch a feature in version 4.7.0, hence this thread.

Thank you for reading and your consideration.

**Comment by tigerkzr** [comment]
+1 seeded time in a relative format (x days yh).  The finished date doesn't quite work because the client might have had some down time since it was finished.
